### DISCUSSION TITLE
Send and reply to messages pattern

### DISCUSSION THREAD

**Discussion by boxadesign** [ONSdigital/design-system#676] [Feedback]
Use this place to discuss the [Send and reply to messages pattern](https://ons-design-system.netlify.com/patterns/send-and-reply-to-messages) in the [ONS Design System](https://ons-design-system.netlify.com)

**Comment by jrbarnes9** [comment]
Focus is lost when traverse between message list items

![Screenshot 2022-08-10 at 18 55 49](https://user-images.githubusercontent.com/43346934/183987154-5044b327-4ba3-442a-b385-ca3dbb495855.png)
